SerialPrograms/Source/PokemonSV/Programs/Sandwiches/PokemonSV_SandwichMaker.cpp

```diff
@@ -53,7 +53,7 @@ SandwichMaker::SandwichMaker()
 void SandwichMaker::program(SingleSwitchProgramEnvironment& env, BotBaseContext& context) {
     assert_16_9_720p_min(env.logger(), env.console);
 
-    run_sandwich_maker(env, context, SANDWICH_OPTIONS);
+    make_sandwich_option(env.program_info(), env.realtime_dispatcher(), env.console, context, SANDWICH_OPTIONS);
```

Comment:
Looks like I overlooked the dispatcher.

Then let's pass `ProgramEnvironment` instead. It's the parent class of Single/MultiSwitchProgramEnvironment and has both the `program_info()` and the dispatchers (as well as the logger).